### PR TITLE
feat(dashboard): clone modal name validation

### PR DESCRIPTION
**Pull request by minseolee**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description

Existing dashboard-clone-modal was markup only. 
So I've improved some codes also .

### CODE REVIEW

File: src/services/dashboards/modules/DashboardCloneModal.vue [hunk 1]
```diff
@@ -121,52 +122,60 @@ export default defineComponent<Props>({
...
+            filteredVisibilityList: computed(() => visibilityList),
+            isProjectDashboard: computed(() => Object.prototype.hasOwnProperty.call(props.dashboard, 'project_id')),
+            dashboardNameList: computed<string[]>(() => {
+                if (state.isProjectDashboard) {
+                    return store.state.dashboard.projectItems
+                        .filter((item) => (item.project_id === (props.dashboard as ProjectDashboardModel).project_id))
+                        .map((_item) => {
+                            if (_item.name !== props.dashboard?.name) return _item.name;
+                            return '';
+                        });
```
Comment (yuda110) [inline src/services/dashboards/modules/DashboardCloneModal.vue:147]: suggestion: how about this?!

```suggestion
                        .filter((item) => (item.project_id === (props.dashboard as ProjectDashboardModel).project_id) && item.name !== props.dashboard.name)
                        .map((_item) => _item.name });
```

Comment (yuda110) [reply src/services/dashboards/modules/DashboardCloneModal.vue:147]: just suggestion! you can ignore it 😅

Comment (minseolee) [reply src/services/dashboards/modules/DashboardCloneModal.vue:147]: Oh I like it !!

File: src/services/dashboards/modules/DashboardCloneModal.vue [hunk 2]
```diff
@@ -121,52 +122,60 @@ export default defineComponent<Props>({
...
-        // });
-
-        const createPublicDashboard = async (): Promise<string|undefined> => {
+        const createDomainDashboard = async (): Promise<string|undefined> => {
             try {
-                // const res = await SpaceConnector.client.costAnalysis.publicDashboard.create(await makeDashboardCreateParam() as DashboardCreateParam);
-                // return res.public_dashboard_id;
+                const res = await SpaceConnector.clientV2.dashboard.domainDashboard.create({
+                    // viewers: viewers.value
+                });
```
Comment (yuda110) [inline src/services/dashboards/modules/DashboardCloneModal.vue:165]: question: is it ok to create without any options?

**Review by yuda110 — APPROVED** [review]
LGTM! I left some comments

File: src/services/dashboards/modules/DashboardCloneModal.vue (hunk 2 above)
Comment (minseolee) [reply src/services/dashboards/modules/DashboardCloneModal.vue:165]: It should be added ! 
I did not checked api working.
I'll add some annotations 
Thank you ~

**Pull request merged**